src/components/Modals/NewCourse.vue

```diff
@@ -1,24 +1,102 @@
 <template>
   <div class="newCourse">
     <div v-if="!isOnboard" class="newCourse-text">{{ text }}</div>
+    <!-- TODO: for some reason this breaks the dropdown <div v-if="selected" class="newCourse-name newCourse-requirements-container">{{ selectedCourse }}</div> -->
     <div class="autocomplete">
       <input class="newCourse-dropdown" :id="'dropdown-' + semesterID" :ref="'dropdown-' + semesterID" :placeholder="placeholder" @keyup.enter="addCourse" @keyup.esc="closeCourseModal" />
     </div>
+    <div v-if="!isOnboard && !selected"> <!-- if a course is not selected -->
+      <div class="newCourse-title">Add this class to the following semester</div>
+      <div class="newCourse-semester-edit">
+        <newSemester :type="season" :year="year"></newSemester>
+      </div>
+    </div>
+    <div v-else > <!-- if a course is selected -->
+      <div class="newCourse-text">Selected Semester</div>
+      <div class="newCourse-semester">
+        <span class="newCourse-name">
+          <img class="newCourse-season-emoji" :src='seasonImg[season]' alt=""> {{ season }} {{ year }}
+        </span>
+      </div>
+      <div class="newCourse-title">This class fulfills the following requirement(s):</div>
+      <div v-if="!editMode" class="newCourse-requirements-container">
+        <div  class="newCourse-requirements" v-for="req in requirements" :key="req" :class="{'newCourse-space': !checkIfLast(req, requirements)}">
```

Comment:
can we also make these requirements editable like the ones below? (i don't think ppl will edit them but we should still give them the option if they want to!)